sources/packages/web/src/components/common/ProgramDetails.vue

```diff
@@ -1,30 +1,32 @@
 <template>
-  <div class="mb-4">
-    <span class="category-header-large color-blue">
-      {{ educationProgram.name }}
-    </span>
-    <status-chip-program
-      class="ml-2 mb-2"
-      :status="educationProgram.programStatus"
-    ></status-chip-program>
-    <v-btn
-      class="float-right"
-      variant="outlined"
-      @click="programButtonAction()"
-      color="primary"
-    >
-      {{ programActionLabel }}
-    </v-btn>
-  </div>
-  <v-row class="secondary-color">
+  <body-header :title="educationProgram.name">
+    <template #status-chip>
+      <status-chip-program
+        class="ml-2 mb-2"
+        :status="educationProgram.programStatus"
+      ></status-chip-program>
+    </template>
+    <template #actions>
+      <v-btn
+        class="float-right label-bold"
+        variant="outlined"
+        @click="programButtonAction()"
+        color="primary"
+      >
+        {{ programActionLabel }}
+      </v-btn>
+    </template>
+  </body-header>
+  <v-row>
     <v-col cols="5">
```

Comment:
As cols is used here, can you try using cols-md and cols-lg as 5 ?

Reply:
why?? I don't think we are planning for Breakpoints for devices in MVP

Reply:
cols are the default (safe if we are using planning for a definite responsiveness for devices)

Reply:
yes, Cols are default so when we set a value to cols IMO it must be set only for the resolutions where we intend to show them in a single row.

Reply:
As discussed, we can discuss the  cols value being set for md, lg and xl with DEVS and make that change in next PR if we all decide to do so.

Reply:
okay, i will add it my next PR